File: 100-exercicios-java/src/main/java/br/com/formulasmatematicas/MediaGeometrica.java
package br.com.formulasmatematicas;

import java.util.Scanner;

/*
 * escreva um programa que calcule a media geometrica
 * entre tres numeros informados pelo usuario
 * 
 * */
public class MediaGeometrica {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		
		System.out.println("Digite o primeiro numero : ");
		double numero1 = scanner.nextDouble();
		
		System.out.println("Digite o segundo numero : ");
		double numero2 = scanner.nextDouble();
		
		System.out.println("Digite o terceiro numero : ");
		double numero3 = scanner.nextDouble();
		
		double mediaGeometrica = Math.pow(numero1 * numero2 * numero3, 1.0/3.0);
		
		System.out.println("A media geometrica dois numeros é "+mediaGeometrica);
		
		scanner.close();
	}
}

File: 100-exercicios-js/formulas-matematicas/media-geometrica.js
/*
  escreva um programa que calcule a media geometrica
  entre tres numeros informados pelo usuario
*/

let num1 = parseFloat(prompt("digite o primeiro numero:"));
let num2 = parseFloat(prompt("digite o segundo numero:"));
let num3 = parseFloat(prompt("digite o terceiro numero:"));

let mediaGeometrica = Math.pow(num1 * num2 * num3, 1/3);

console.log("a media geometrica dos numeros é :", mediaGeometrica)